go run -tags=mobile panic

Go version:

```
$ go version
go version go1.14 darwin/amd64
```

Code:

```Go
package main

import (
	"image/color"

	"fyne.io/fyne"
	"fyne.io/fyne/app"
	"fyne.io/fyne/canvas"
)

func main() {
	app := app.New()

	w := app.NewWindow("hello")
	w.Resize(fyne.NewSize(400, 300))

	w.SetContent(canvas.NewRectangle(color.RGBA{R: 255, A: 255}))

	w.ShowAndRun()
}
```

Go.mod:

```
module hello-fyne

go 1.14

require (
	fyne.io/fyne v1.2.3
	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
)
```

run:

```
$ go run -tags=mobile .
# golang.org/x/mobile/app
darwin_desktop.m:51:1: warning: method possibly missing a [super prepareOpenGL] call [-Wobjc-missing-super-calls]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x4302d5c]

goroutine 1 [running, locked to thread]:
fyne.io/fyne/internal/driver/gomobile.(*mobileCanvas).sizeContent(0xc000200000, 0x190, 0x12c)
	/Users/chai/go/pkg/mod/fyne.io/fyne@v1.2.3/internal/driver/gomobile/canvas.go:79 +0x7c
fyne.io/fyne/internal/driver/gomobile.(*mobileCanvas).resize(...)
	/Users/chai/go/pkg/mod/fyne.io/fyne@v1.2.3/internal/driver/gomobile/canvas.go:92
fyne.io/fyne/internal/driver/gomobile.(*window).Resize(0xc0001083c0, 0x190, 0x12c)
	/Users/chai/go/pkg/mod/fyne.io/fyne@v1.2.3/internal/driver/gomobile/window.go:40 +0x59
main.main()
	/Users/chai/play/hello-fyne/main.go:15 +0xa3
exit status 2
```

--- a/internal/driver/gomobile/driver.go
+++ b/internal/driver/gomobile/driver.go
@@ -6,6 +6,7 @@ import (
 	"strconv"
 	"time"
 
+	"fyne.io/fyne/canvas"
 	"fyne.io/fyne/internal"
 	"fyne.io/fyne/widget"
 	"golang.org/x/mobile/app"
@@ -41,10 +42,10 @@ func init() {
 }
 
 func (d *mobileDriver) CreateWindow(title string) fyne.Window {
-	canvas := NewCanvas().(*mobileCanvas) // silence lint
-	ret := &window{title: title, canvas: canvas, isChild: len(d.windows) > 0}
-	canvas.painter = pgl.NewPainter(canvas, ret)
-
+	c := NewCanvas().(*mobileCanvas) // silence lint
+	ret := &window{title: title, canvas: c, isChild: len(d.windows) > 0}
+	c.content = &canvas.Rectangle{FillColor: theme.BackgroundColor()}
+	c.painter = pgl.NewPainter(c, ret)
 	d.windows = append(d.windows, ret)
 	return ret
 }

